src/Employer/Employer.Web/Orchestrators/ManageNotificationsOrchestrator.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Threading.Tasks;
+using Esfa.Recruit.Employer.Web.ViewModels.ManageNotifications;
+using Esfa.Recruit.Shared.Web.Orchestrators;
+using Esfa.Recruit.Vacancies.Client.Application.Validation;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+using Microsoft.Extensions.Logging;
+
+namespace Esfa.Recruit.Employer.Web.Orchestrators 
+{
+    public class ManageNotificationsOrchestrator : EntityValidatingOrchestrator<UserNotificationPreferences, ManageNotificationsEditModel>
+    {
+        private readonly IRecruitVacancyClient _recruitVacancyClient;
+
+        private const string NotificationTypesIsRequiredForTheFirstTime = "Choose when you’d like to receive emails";
+        private readonly EntityValidationResult _notificationTypeIsRequiredForTheFirstTime = new EntityValidationResult
+        {
+            Errors = new[] 
+            { 
+                new EntityValidationError(1100, nameof(ManageNotificationsEditModel.HasAnySubscription), NotificationTypesIsRequiredForTheFirstTime, "1100") 
```

Comment:
Can the 1100 be put in a const please.

Reply:
It is of no value what so ever, don't think every literal needs to be a constant. It makes sense where you can give it a sensible name and make the code more readable which is not the case here.